src/styles/public/print.scss

```diff
@@ -18,9 +24,9 @@
 		* {
 			all: revert;
 			aspect-ratio: unset !important;
-			color: #333 !important;
 			background: none !important;
 			border: none !important;
+			color: #333 !important;
```

Comment:
just curious why this color was used #333, design system is #313132 https://developer.gov.bc.ca/Design-System/Colour-Palette

Reply:
For print I prefer using a soft black that remains fully neutral.